website_snippet_product_category/static/src/scss/snippet.scss

```diff
@@ -0,0 +1,42 @@
+.categ_container {
+    border: 0 solid $border-color;
+    border-width: 0 1px 0 0;
+
+    .categ_tree_level {
+        &.main_tree_level {
+            color: $primary;
+            text-transform: uppercase;
+        }
+
+        &[data-tree-level="2"] {
+            font-size: 0.9em;
```

Comment:
:no_entry_sign: Always use `rem` units: https://getbootstrap.com/docs/4.0/migration/#global-changes

Reply:
Umm in this place i prefer use "em" instead of "rem"... if you change the font-size of 'categ-container' you can see how all the other texts are "adapted" to this "parent" rule.

https://www.w3.org/Style/Examples/007/units.en.html#font-size